pkg/operator/controller.go

```diff
@@ -131,7 +131,7 @@ func (c *Controller) sync() error {
 		proxyConfig = &configapiv1.Proxy{}
 	} else if err != nil {
 		// TODO: should we report Degraded?
-		return fmt.Errorf("unable to get cluster proxy configuration: %v", err)
+		glog.Warningf("unable to get cluster proxy configuration: %v", err)
```

Comment:
we need to sort out why the existing "IsNotFound" check didn't catch this scenario and revisit what the right thing to do is here, but for now this hopefully gets us back to green on the release jobs.

Reply:
Because it's a wrong place... The new error happens there:

https://github.com/openshift/cluster-image-registry-operator/blob/f10bd546a46c3a15db8c760be9c80295808f78d4/pkg/resource/podtemplatespec.go#L116-L119

Reply:
Yep, just found that, i have a pr up in a second